### PR TITLE
Added "Send OTP over USB" button and related UX adjustments

### PR DESCRIPTION
**Pull request by geco2**
it does not look better than before... But at least usable for me, wonder if others like it as well.

### CODE REVIEW

File: authorizer/src/main/java/net/tjado/passwdsafe/PasswdSafeRecordBasicFragment.java [hunk 1]
```diff
@@ -702,17 +724,32 @@ private void updatePasswordShown(boolean isToggle, int progress)
...
+            if( sendOTP == true && otp != null ) {
+                generateOtpToken();
+                otpTokenGenerated = true;
+
+                int ret = 0;
+                ret = ct.sendText(itsOtp.getCurrentCode());
+
+                if (ret == 1) {
+                    PasswdSafeUtil.showErrorMsg(
+                            "Unvalid OTP token generated!",
```
Comment (tejado) [inline authorizer/src/main/java/net/tjado/passwdsafe/PasswdSafeRecordBasicFragment.java:748]: "Unvalid" Typo

Comment (geco2) [reply authorizer/src/main/java/net/tjado/passwdsafe/PasswdSafeRecordBasicFragment.java:748]: thank you!

**Review by tejado — COMMENTED** [review]
Hi Andi,

thanks a lot for your PR. Really good!
Would be great if you can adjust the travis ci file (.travis.yml in /) as the build was not successful due to sdk version change. But if you like, I can also do that later.

And one additional hint for the future: it is normally better to make one change in one commit so in case of a rollback it can be done more easily (in this PR it can be done in three commits: gradle update, sdk version change and then the adjustments of the UI including the Send OTP button). Sometimes it makes also sense to make an own PR for it so that a discussion of one change don't stop the merge of the other ones.

**Comment by geco2** [comment]
yes, indeed a stupid idea -_-

Sorry, i'm simply not realized how the update of my AndroidStudio influenced the pr

**Comment by geco2** [comment]
Seems the android.support libraries will be replaced by AndroidX (https://developer.android.com/jetpack/androidx/) which makes my PR exploding cause that comes with a migration for the whole project :-D and there is no pack port for test:runner and others for sdk-version 28. Hope i'm wrong...

File: lib/build.gradle [hunk 2]
```diff
@@ -24,8 +24,8 @@ def buildTime() {
 }
 
 android {
-    compileSdkVersion 27
-    buildToolsVersion '27.0.3'
+    compileSdkVersion 28
+    buildToolsVersion '28.0.3'
```
Comment (tejado) [inline lib/build.gradle]: Can you revert these both lines? Then travis build should work

**Pull request merged**